mindsdb/integrations/handlers/statsforecast_handler/requirements.txt

```diff
@@ -1 +1,4 @@
-statsforecast>=1.4.0, <2.0
\ No newline at end of file
+statsforecast>=1.5.0, <2.0
+neuralforecast>=1.4.0, <2.0
+hierarchicalforecast<1.0
+datasetsforecast<1.0
```

Comment:
Is this actually needed? Can't find any references in the rest of the PR

Reply:
Sorry, missed this one! It's not needed any longer, it was a hangover from importing test data. Have removed now (except for HF which is needed).